dev/io.openliberty.microprofile.telemetry.spi/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion: 1.0
+
+Bundle-Name: Liberty MicroProfile Telemetry SPI
+Bundle-Description: Liberty MicroProfile Telemetry SPI, version ${bVersion}
+Bundle-SymbolicName: io.openliberty.microprofile.telemetry.spi
+
+Import-Package: io.openliberty.microprofile.telemetry.spi
+
+Export-Package: io.openliberty.microprofile.telemetry.spi
+
+-maven-dependencies:
+ 
+publish.wlp.jar.suffix: dev/spi/ibm
```

Comment:
If you are going to actually have this be beta, it probably should be put into the lib directory initially until it is marked GA.